app/providers/operations/conversation/delete/index.js

```diff
@@ -14,9 +14,19 @@ class ConversationDeleteOperation {
 
     const { conversation, participantIds } = await this.#hasAccess(conversationId, currentUserId)
```

Comment:
`this.#hasAccess` means the function should return true or false, but it returns object.

Let's renamed to `#getConversationDetails`